### PR TITLE
Docker (PR #5502 follow up)

### PR DESCRIPTION
**Pull request by charlesfleche**
Follows up #5502

This PR deactivate JAR signing, add a new commcare-hq environment variable BASE_HOST to set Django's BASE_NAME, add a docker README and exclude localsettings.py from lint checks.

### CODE REVIEW

File: docker/localsettings.py [hunk 1]
```diff
@@ -77,14 +77,14 @@
...
 
-_ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
-JAR_SIGN = {
-    'jad_tool': os.path.join(_ROOT_DIR, "corehq", "apps", "app_manager", "JadTool.jar"),
-    'key_store': os.path.join(os.path.dirname(os.path.dirname(_ROOT_DIR)), "DimagiKeyStore"),
-    'key_alias': "javarosakey",
-    'store_pass': "*******",
-    'key_pass': "*******",
-}
+#_ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```
Comment (dimagimon) [inline docker/localsettings.py]: E265 block comment should start with '# '

File: docker/localsettings.py [hunk 2]
```diff
@@ -77,14 +77,14 @@
...
+#JAR_SIGN = {
```
Comment (dimagimon) [inline docker/localsettings.py]: E265 block comment should start with '# '

File: docker/localsettings.py [hunk 3]
```diff
@@ -77,14 +77,14 @@
...
+#    'jad_tool': os.path.join(_ROOT_DIR, "corehq", "apps", "app_manager", "JadTool.jar"),
+#    'key_store': os.path.join(os.path.dirname(os.path.dirname(_ROOT_DIR)), "DimagiKeyStore"),
+#    'key_alias': "javarosakey",
+#    'store_pass': "*******",
+#    'key_pass': "*******",
+#}
```
Comment (dimagimon) [inline docker/localsettings.py]: E265 block comment should start with '# '

File: docker/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* Wheezy has been choosen as a base image because the official redis, postgres and couchdb are based on the same image, potentialy reducing the amount of data to download.
+* Databases initializations (users, db, django) is done by instanciating temporary couchdb / postgres / commcarehq images that already contain the necessary binaries (like psql).
+* Celery, rabbitmq and other components not strictly necessary for a laptop install are not part of this setup.
+
+
+Caveats
+-------
+
+* CloudCare is not currently part of this set up. It should probably be another docker image, different from CommCareHQ.
+* Some CommCareHQ pages trigger errors, probably because of missing components / misconfigurations.
```
Comment (snopoke) [inline docker/README.md]: Should add that export / import etc won't work since celery is also not configured.

Comment (charlesfleche) [inline docker/README.md]: Are you talking about case bulk upload ?

Comment (snopoke) [inline docker/README.md]: Yes

**Pull request merged**